v2/internal/pkg/operator/filtered_collector.go

```diff
@@ -49,6 +49,7 @@ func (o *FilterCollector) OperatorImageCollector(ctx context.Context) (v2alpha1.
 	collectorSchema := v2alpha1.CollectorSchema{}
 	copyImageSchemaMap := &v2alpha1.CopyImageSchemaMap{OperatorsByImage: make(map[string]map[string]struct{}), BundlesByImage: make(map[string]map[string]string)}
 
+	var errs []error
```

Comment:
Nit: since this is not being used in this PR, we can remove it and introduce later. But no harm in keeping it if you prefer.